src/components/filtered.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import '../styles/filtered.css';
+import { useSelector } from 'react-redux';
+import Card from './card';
+
+const Filtered = () => {
+  const { airdata } = useSelector((state) => state.airqa);
+  const [filt, setFilt] = useState([]);
+  const [categ, setCateg] = useState('');
+
+  const handleCatChng = (e) => {
+    setCateg(e.target.value);
+  };
+  const filterArray = () => {
+    if (categ === 'best') {
+      const newArray = airdata.filter((ele) => ele.aqi <= 2);
+      setFilt(newArray);
+    } else if (categ === 'worst') {
+      const newArray = airdata.filter((ele) => ele.aqi >= 3);
+      setFilt(newArray);
+    } else if (categ === 'all') {
+      setFilt(airdata);
+    }
+  };
+
+  return (
+    <>
+      <div className="filtered">
+        <form action="POST">
+          <select name="filter" id="form-filter" onChange={handleCatChng}>
+            <option value="">-Please choose an option-</option>
+            <option value="worst">Worst</option>
+            <option value="best">Best</option>
+            <option value="all">All</option>
+          </select>
+          <br />
+          <button type="button" onClick={filterArray}>Filter</button>
+        </form>
+      </div>
+      <ul className="card-container">
+        {filt.map((ele) => (
+          <Card key={ele.lat} data={ele} />
+        ))}
+      </ul>
+    </>
+
+  );
+};
+
```

Comment:
- [ ] You did well implementing this requirement by the previous reviewer. I wish he answered your question because he would have told you that you do not need to make a separate page just for the filter feature. The filter feature should be just below the navbar and just above the category list. See this 👇🏾 should be your default page so that the user can filter from here. Notice that in this case, the filter input will have no value on page load. 

![image](https://github.com/PabloEGonz/air-quality-app/assets/84156714/7f09cd8a-5876-4f15-9f8c-97c1d1e55e2b)